odl/trafos/wavelet.py

```diff
@@ -571,6 +872,11 @@ def __init__(self, dom, nscales, wbasis, mode):
 
             'per': periodization -- like periodic-padding but gives the
             smallest possible number of decomposition coefficients.
+        axes : sequence of `int`, optional
+            Dimensions in which to calculate the wavelet transform.
+            The sequence's length has to be equal to dimension of the ``grid``
+            `None` means traditional transform along the axes in ``grid``.
+            If axes is given nscales is discarded.
```

Comment:
Also write parameter names with ticks.

If \`\`axes\`\` is given \`\`nscales\`\` is discarded.